From ba8376e8c76c6512c98949dffbf4ded0b782a365 Mon Sep 17 00:00:00 2001
From: Manivannan Sadhasivam <mani@kernel.org>
Date: Tue, 11 Aug 2026 16:34:17 +0200
Subject: [PATCH 13307/15333] MAINTAINERS: Add Manivannan Sadhasivam as the
 Reviewer for Generic PHY Framework

I'm volunteering to review the PHY patches. Hence, add myself as the
Reviewer.

Signed-off-by: Manivannan Sadhasivam <mani@kernel.org>
Link: https://patch.msgid.link/20260811143417.336664-1-mani@kernel.org
Signed-off-by: Vinod Koul <vkoul@kernel.org>
---
 MAINTAINERS | 1 +
 1 file changed, 1 insertion(+)

diff --git a/MAINTAINERS b/MAINTAINERS
index 5b544583e4d8..80c0cf242887 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -10920,6 +10920,7 @@ F:	include/uapi/asm-generic/
 GENERIC PHY FRAMEWORK
 M:	Vinod Koul <vkoul@kernel.org>
 R:	Neil Armstrong <neil.armstrong@linaro.org>
+R:	Manivannan Sadhasivam <mani@kernel.org>
 L:	linux-phy@lists.infradead.org
 S:	Supported
 Q:	https://patchwork.kernel.org/project/linux-phy/list/
-- 
2.55.0

